src/heretic/d_net.c

```diff
@@ -142,7 +149,9 @@ static void SaveGameSettings(net_gamesettings_t *settings)
     settings->nomonsters = nomonsters;
     settings->respawn_monsters = respawnparm;
     settings->timelimit = 0;
-    settings->lowres_turn = false;
+
+    settings->lowres_turn = M_ParmExists("-record")
+                        && !M_ParmExists("-longtics");
```

Comment:
My bad! I look at this and say "it's aligned because the functions line up". I assume you want this?
```
settings->lowres_turn = M_ParmExists("-record")
                     && !M_ParmExists("-longtics");